content/docs/start/data-pipelines.md

```diff
@@ -304,23 +306,23 @@ stages:
 DVC pipelines (`dvc.yaml` file, `dvc run`, and `dvc repro` commands) solve a few
 important problems:
 
-- _Automation_ - run a sequence of steps in a "smart" way that makes iterating
+- _Automation_: run a sequence of steps in a "smart" way which makes iterating
   on your project faster. DVC automatically determines which parts of a project
-  need to be run, and it caches "runs" and their results, to avoid unnecessary
-  re-runs.
-- _Reproducibility_ - `dvc.yaml` and `dvc.lock` files describe what data to use
+  need to be run, and it caches "runs" and their results (in the aptly-named
+  <abbr>run-cache</abbr>) to avoid unnecessary reruns.
+- _Reproducibility_: `dvc.yaml` and `dvc.lock` files describe what data to use
   and which commands will generate the pipeline results (such as an ML model).
   Storing these files in Git makes it easy to version and share.
-- _Continuous Delivery and Continuous Integration (CI/CD) for ML_ - describing
-  projects in way that it can be reproduced (built) is the first necessary step
-  before introducing CI/CD systems. See our sister project,
+- _Continuous Delivery and Continuous Integration (CI/CD) for ML_: describing
+  projects in way which it can be reproduced (built) is the first necessary step
+  before introducing CI/CD systems. See our sister project
   [CML](https://cml.dev/) for some examples.
 
 ## Visualize
 
 Having built our pipeline, we need a good way to understand its structure.
-Seeing a graph of connected stages would help. DVC lets you do just that,
-without leaving the terminal!
+Seeing a graph of connected stages would help. DVC lets you do just that without
+leaving the terminal!
```

Comment:
Is "do just that without" readable without comma? Maybe without "just"

Reply:
No essentially another comma splice. Pauses when speaking do not have to appear when reading. Could use ellipsis instead of comma here to force a pause if you want.

Reply:
> Pauses when speaking do not have to appear when reading.

We may overuse them a little 🙂 But in this case it seems useful. The problem for me here is "do just that without". So maybe "do so without"?

Reply:
done